data/human/news.txt

```diff
@@ -1963,6 +1995,33 @@ news "republic navy advisory system [wartime]"
 		word
 			`"`
 
+news "explorer [frozen]"
+	location
+		government "Republic" "Free Worlds" "Neutral" "Independent" "Syndicate"
+		attributes "frozen"
+	name
+		word
+			"Arctic"
+			"Glacial"
+		word
+			"explorer"
+			"surveyor"
+	message
+		word
+			`"`
+		word
+			"There's only so much you can get from orbital mapping. Sometimes, you've got to gear up and go see it yourself."
+			"Cold? You don't know cold. You don't know true cold until you've been bivouacked in a scree field for a week, wondering when the storm will end."
+			"Happiness is finding a can of peaches you buried in the snow a month ago."
+			"It's my responsibility to check on the remote weather stations. If one of those go down, the whole colony could be at risk."
+			"There's no bad weather, only bad clothing."
+			"It lacks the romance of furs and seal skin, but I'm glad for modern textiles."
+			"It's hard to have a bond with a snowcat quite like you do with a husky."
```

Comment:
Might not matter, but are there snowcats on other planets?

Reply:
"[Snowcat](https://en.wikipedia.org/wiki/Snowcat)" is a deliberate play on words vis-a-vis a dog (husky) as arctic transport.

If the joke is too niche, I'm happy removing it, though.

Reply:
I did not know what that is. I would wait for other people's input, though.

Reply:
No no, leave it.